src/components/editor/State.js

```diff
@@ -1262,43 +1257,87 @@ class Observation extends Component<Props> {
       );
     } else if (state.attribute) {
       return (
         <div className='section'>
           Attribute: <RIEInput className='editable-text' value={state.attribute} propName={'attribute'}  change={this.props.onChange('attribute')} />
           <br/>
-          <a className='editable-text' onClick={() => {this.props.onChange('exact')({val: {id: getTemplate('Attribute.Exact')}}); this.props.onChange('range')({val: {id: null}}); this.props.onChange('attribute')({val: {id: null}}); this.props.onChange('vital_sign')({val: {id: null}})}}>Change to Exact</a>
-          <br />
-          <a className='editable-text' onClick={() => {this.props.onChange('range')({val: {id: getTemplate('Attribute.Range')}}); this.props.onChange('exact')({val: {id: null}}); this.props.onChange('attribute')({val: {id: null}}); this.props.onChange('vital_sign')({val: {id: null}})}}>Change to Range</a>
-          <br />
-          <a className='editable-text' onClick={() => {this.props.onChange('vital_sign')({val: {id: "text"}}); this.props.onChange('exact')({val: {id: null}}); this.props.onChange('range')({val: {id: null}}); this.props.onChange('attribute')({val: {id: null}})}}>Change to Vital Sign</a>
-          <br />
+          { this.renderToggles('attribute') }
+        </div>
+      );
+    } else if (state.value_code) {
+      return (
+        <div className='section'>
+          <Code code={state.value_code} system={"SNOMED-CT"} onChange={this.props.onChange('value_code')} />
+          { this.renderToggles('value_code') }
         </div>
       );
     } else { // vital_sign
       return (
         <div className='section'>
           Vital Sign: <RIEInput className='editable-text' value={state.vital_sign} propName={'vital_sign'}  change={this.props.onChange('vital_sign')} />
-          <br/>
-          <a className='editable-text' onClick={() => {this.props.onChange('exact')({val: {id: getTemplate('Attribute.Exact')}}); this.props.onChange('range')({val: {id: null}}); this.props.onChange('attribute')({val: {id: null}}); this.props.onChange('vital_sign')({val: {id: null}})}}>Change to Exact</a>
-          <br />
-          <a className='editable-text' onClick={() => {this.props.onChange('range')({val: {id: getTemplate('Attribute.Range')}}); this.props.onChange('exact')({val: {id: null}}); this.props.onChange('attribute')({val: {id: null}}); this.props.onChange('vital_sign')({val: {id: null}})}}>Change to Range</a>
-          <br />
-          <a className='editable-text' onClick={() => {this.props.onChange('attribute')({val: {id: "text"}}); this.props.onChange('exact')({val: {id: null}}); this.props.onChange('range')({val: {id: null}}); this.props.onChange('vital_sign')({val: {id: null}})}}>Change to Attribute</a>
-          <br />
+          { this.renderToggles('vital_sign') }
         </div>
       );
     }
   }
 
+  renderToggles(currentItem) {
+    console.log(currentItem);
+
+    let toggles = [];
+
+    if (currentItem != 'exact') {
```

Comment:
Minor issue: this should be `!==`.  Also in a few more places below.  React is throwing a warning on these.  I realize there are a handful of other places in the codebase with this now, but may as well not introduce any more.